### PR TITLE
i18n(pt-BR): Adding `editor-setup.md` & `integrations.md` translations + `ui.ts` update

### PR DESCRIPTION
**Pull request by Yan-Thomas**
Translations for `editor-setup.md` and `integrations.md`. Also updated `ui.ts` with the "Astro Docs on GitHub" translation.

### CODE REVIEW

File: src/pages/pt-BR/editor-setup.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Configuração do Editor
+description: Configure seu editor para desenvolver com Astro.
+i18nReady: true
+---
+
+Customize seu editor de código para melhorar a sua experiência de desenvolvimento com Astro e desfrute de novas funcionalidades.
+
+## VS Code
+
+[VS Code](https://code.visualstudio.com) é um popular editor de código para desenvolvedores web, feito pela Microsoft. O motor do VS Code também viabiliza populares editores de código no navegador como o [GitHub Codespaces](https://github.com/features/codespaces) e o [Gitpod](https://gitpod.io).
```
Comment (caioferrarezi) [inline src/pages/pt-BR/editor-setup.md:14]: I think I would change the adjective position here:
```suggestion
[VS Code](https://code.visualstudio.com) é um popular editor de código para desenvolvedores web, feito pela Microsoft. O motor do VS Code também viabiliza editores de código populares no navegador como o [GitHub Codespaces](https://github.com/features/codespaces) e o [Gitpod](https://gitpod.io).
```

Comment (Yan-Thomas) [reply src/pages/pt-BR/editor-setup.md:14]: Thanks!

File: src/pages/pt-BR/integrations/integrations.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Feito com Astro
+i18nReady: true
+---
+
+Membros da comunidade Astro tem com sucesso integrado serviços de terceiros em seus websites Astro!
```
Comment (caioferrarezi) [inline src/pages/pt-BR/integrations/integrations.md:7]: same here, I think this make more sense in english but feels unnatural in portuguese:
```suggestion
Membros da comunidade Astro tem integrado com sucesso os serviços de terceiros em seus websites Astro!
```

Comment (Yan-Thomas) [reply src/pages/pt-BR/integrations/integrations.md:7]: I agree, thanks!

**Review by caioferrarezi — APPROVED** [review]
LGTM! 🚀

**Pull request merged**